test/EleventyFilesGitIgnoreEleventyIgnoreTest.js

```diff
@@ -193,45 +193,6 @@ test("Get ignores (both .eleventyignore and .gitignore exists, but .gitignore is
   ]);
 });
 
-test("Bad expected output, this indicates a bug upstream in a dependency (update, was fixed in fast-glob@3.3.3).  Input to 'src' and empty includes dir (issue #403, full paths in eleventyignore)", async (t) => {
-	let eleventyConfig = await getTemplateConfigInstanceCustomCallback({
-		input: "test/stubs-403",
-		output: "_site",
-		includes: "",
-		data: false,
-  }, function(eleventyConfig) {
-    eleventyConfig.setUseGitIgnore(false);
-  });
-
-  let { eleventyFiles: evf } = getEleventyFilesInstance(["liquid"], eleventyConfig);
-  evf._setEleventyIgnoreContent("!" + TemplatePath.absolutePath("test/stubs-403/_includes") + "/**");
-  evf.init(); // duplicate init
-
-  t.deepEqual(await evf.getFiles(), [
-    "./test/stubs-403/template.liquid",
-    // UPDATE: this was fixed in fast-glob@3.3.3
-    // This should be excluded from this list but is not because the ignore content used an absolutePath above.
-    // "./test/stubs-403/_includes/include.liquid",
-  ]);
-});
-
-test("Workaround for Bad expected output, this indicates a bug upstream in a dependency.  Input to 'src' and empty includes dir (issue #403, full paths in eleventyignore)", async (t) => {
-	let eleventyConfig = await getTemplateConfigInstanceCustomCallback({
-		input: "test/stubs-403",
-		output: "_site",
-		includes: "",
-		data: false,
-  }, function(eleventyConfig) {
-    eleventyConfig.setUseGitIgnore(false);
-  });
-
-  let { eleventyFiles: evf } = getEleventyFilesInstance(["liquid"], eleventyConfig);
-  evf._setEleventyIgnoreContent("!./test/stubs-403/_includes/**");
-  evf.init(); // duplicate init
-
-  t.deepEqual(await evf.getFiles(), ["./test/stubs-403/template.liquid"]);
-});
-
```

Comment:
I removed these tests as from what I gather they were related to a bug in an old `fast-glob` version. Down to add them back as a sanity check though.

Reply:
I restored and fixed these tests